spacy/ml/models/tok2vec.py

```diff
@@ -118,7 +117,7 @@ def hash_embed_bilstm_v1(
 
 @registry.architectures.register("spacy.HashCharEmbedBiLSTM.v1")
 def hash_char_embed_bilstm_v1(
-    pretrained_vectors, width, depth, embed_size, subword_features, nM, nC, maxout_pieces
+    pretrained_vectors, width, depth, embed_size, maxout_pieces, nM=0, nC=0
```

Comment:
I don't think we should have default values here? I thought the point was to have it all in the configuration file, so there are no "hidden" defaults set in potentially different places anymore.